x-pack/heartbeat/monitors/browser/browser.go

```diff
@@ -52,16 +52,23 @@ func create(name string, cfg *common.Config) (p plugin.Plugin, err error) {
 		return plugin.Plugin{}, err
 	}
 
+	extraArgs := []string{}
+	if ss.suiteCfg.Sandbox {
+		extraArgs = append(extraArgs, "--sandbox")
+	} else {
+		extraArgs = append(extraArgs, "--no-sandbox")
```

Comment:
we should keep this as it is and not add any args.

Reply:
As discussed I removed this, there's now only a `--sandbox` option since synthetics will default to sandbox off.